ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/CardinalClient.kt

```diff
@@ -109,6 +109,9 @@ internal class CardinalClient {
             cardinalConfigurationParameters.uiCustomization = it.cardinalUiCustomization
         }
 
+        request.requestorAppUrl?.let {
+            cardinalConfigurationParameters.threeDSRequestorAppURL = request.requestorAppUrl
```

Comment:
```suggestion
            cardinalConfigurationParameters.threeDSRequestorAppURL = it
```

We can reference `it` here within the `let` scoping function, which will have a non-null type. `threeDSRequestorAppURL` probably handles the null case, but using `it` here will prevent us from ever passing a null value, since `requestorAppUrl` could change between the null check and referencing the value.

Reply:
Scope functions can get really confusing in Kotlin 🥴 Here's a good doc page on them that I'll frequently reference for the Function Selection table: https://kotlinlang.org/docs/scope-functions.html

Reply:
This doc is super helpful! Thanks for sharing, I'll get there on Android/Kotlin eventually. 😅